Documentation/Blazorise.Docs/Pages/Docs/Extensions/TreeView/TreeViewPage.razor

```diff
@@ -192,6 +192,9 @@
     <DocsAttributesItem Name="AutoExpandAll" Type="bool" Default="false">
         Defines if the treenode should be automatically expanded. Note that it can happen only once when the tree is first loaded.
     </DocsAttributesItem>
+    <DocsAttributesItem Name="Virtualize" Type="bool">
+        Defines if hidden treenodes should be visible. See <a href="https://learn.microsoft.com/en-us/aspnet/core/blazor/components/virtualization">docs for Virtualization</a>.
```

Comment:
Use our `Anchor` component for links. 

Example: `<Anchor To="https://learn.microsoft.com/en-us/aspnet/core/blazor/components/virtualization">docs for Virtualization</Anchor>`